content/guide/printing/index.um

```diff
@@ -1,44 +1,70 @@
-@inline ../../templates/sidebar-page.um
+@inline /templates/sidebar-page.um
```

Comment:
Updated the printing page to use new quantum format instead of manual `p` and `div` elements